tests/test_auto_int.py

```diff
@@ -0,0 +1,26 @@
+from .base import BaseTestCase
+from subconscious.column import Integer, Column
+from subconscious.model import RedisModel
+
+
+class TestUser(RedisModel):
+    id = Integer(primary_key=True, auto=True)
+    name = Column(type=str)
+    auto_id = Integer(auto=True)
+
+
+class TestAutoInt(BaseTestCase):
+
+    def test_auto(self):
+        user = TestUser(name='foo')
+        self.loop.run_until_complete(user.save(self.db))
+        self.assertEqual(user.id, 1)
+        user = TestUser()
+        self.loop.run_until_complete(user.save(self.db))
+        self.assertEqual(user.id, 2)
+
+        # Provide id
+        user = TestUser(id=777)
+        self.loop.run_until_complete(user.save(self.db))
+        self.assertEqual(user.id, 777)
```

Comment:
This seems like a bug, as this column also has `auto=True`. This should either = 3 or raise an exception.